app/src/main/res/layout/multiple_choice_interaction_items.xml

```diff
@@ -39,6 +39,7 @@
       android:layout_toEndOf="@+id/multiple_choice_radio_button"
       android:fontFamily="sans-serif"
       android:paddingTop="4dp"
+      android:paddingBottom="4dp"
```

Comment:
**Comment 2**
This paddingTop and paddingBottom has been introduced because the `AppCompatRadioBottom` also contains paddingTop and paddingBottom of `4dp` by default.